modin/pandas/test/internals/test_benchmark_mode.py

```diff
@@ -15,17 +15,38 @@
 
 from contextlib import nullcontext
 import modin.pandas as pd
-from modin.pandas.test.utils import test_data_values
+from modin.pandas.test.utils import test_data_values, df_equals
 from modin.config import BenchmarkMode, StorageFormat
 from modin.test.test_utils import warns_that_defaulting_to_pandas
 
 
+default_to_pandas_context = (
+    warns_that_defaulting_to_pandas()
+    if StorageFormat.get() == "Omnisci"
+    else nullcontext()
+)
+
+
 def test_syncronous_mode():
     assert BenchmarkMode.get()
     # On Omnisci storage, transpose() defaults to Pandas.
-    with (
-        warns_that_defaulting_to_pandas()
-        if StorageFormat.get() == "Omnisci"
-        else nullcontext()
-    ):
+    with default_to_pandas_context:
         pd.DataFrame(test_data_values[0]).mean()
+
+
+def test_serialization():
+    assert BenchmarkMode.get()
+
+    sr1 = pd.Series(range(10))
```

Comment:
Maybe parametrize data via pytest?

Reply:
what for? the test doesn't depend on the data provided, it always works the same

Reply:
[Suggestion.](https://github.com/modin-project/modin/pull/4404/files#r861714759)